hypervisor/include/lib/bits.h

```diff
@@ -169,6 +172,22 @@ static inline uint16_t clz64(uint64_t value)
 	}
 }
 
+static inline uint64_t ffz64_ex(const uint64_t *addr, uint64_t size)
+{
+	uint64_t idx;
+
+	if ((size & (BITS_PER_LONG - 1UL)) != 0UL) {
+		return size;
+	}
```

Comment:
We can also go with pre-assumption of the parameter.. WHich one is better?

Reply:
IMO, it should be the lib function itself to make sure the input parameters are valid.